src/Components/Results/ResultsTitle/index.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import './ResultsTitle.css'
+
+const ResultsTitle = () => <h1>THE ALL-TIME TOP 10!</h1>
```

Comment:
How about a re-usable title component that gets given the text as a prop instead of the two individual titles?